lib/src/facts/osx/operating_system_resolver.cc

```diff
@@ -33,8 +36,21 @@ namespace facter { namespace facts { namespace osx {
                 result.osx.version = move(value);
             }
 
+            // Populate the major and minor version facts using kernel_release
+            auto release = facts.get<string_value>(fact::kernel_release);
```

Comment:
@peterhuene Doh, at first I thought I was going to be using sw_vers to get the version but then realized it had to come from the kernel facts, and just forgot to move this bit out of the lambda :P